content/authors/stuara/_index.md

```diff
@@ -58,20 +68,22 @@ social:
 # - icon: github
 #   icon_pack: fab
 #   link: https://github.com/imranmaj
-# - icon: linkedin
-#   icon_pack: fab
-#   link: https://www.linkedin.com/in/imranmaj/
 # Link to a PDF of your resume/CV from the About widget.
 # To enable, copy your resume/CV to `static/files/cv.pdf` and uncomment the lines below.
 # - icon: cv
 #   icon_pack: ai
 #   link: files/cv.pdf
 
 # Enter email to display Gravatar (if Gravatar enabled in Config)
-email: ""
+# email: ""
 
 # Organizational groups that you belong to (for People widget)
 #   Set this to `[]` or comment out if you are not using People widget.
 user_groups:
 - Researchers
 ---
+I am an undergrad at the University of Washington studying Marine Biology and Art. I intend to graduate in the Spring of 2023. I hope to pursue projects where both scientific and artistic thinking can compliment each other.
+
+With Echospace, I am assisting with the creation of programs used to analyze and compress large volumes of biologically-oriented acoustic data, including that of the NCWFSC Pacific hake survey. I will be further developing my preexisting programming skills to better benefit this and future projects.
+
+I am also participating in the collection and analysis of passive acoustic data targeting native bat species from the UW Union Bay Natural Area , as well the operation and deployment of ADCPs off the Washington Coastal Shelf.
```

Comment:
suggested edits:
- extra space after "Area"
- as well *as* the operation...
- spell out ADCP so that people know what it is
- probably better to say ADCP and an ADCP-equipped glider